sinatra.gemspec

```diff
@@ -46,7 +46,7 @@ RubyGems 2.0 or newer is required to protect against public gem pushes. You can
   s.required_ruby_version = '>= 2.6.0'
 
   s.add_dependency 'mustermann', '~> 3.0'
-  s.add_dependency 'rack', '~> 2.2', '>= 2.2.4'
+  s.add_dependency 'rack', '>= 2.2.4'
```

Comment:
Is it possible to not make this change now? Not sure how besides not loading gemspec in Gemfile and repeat our dependencies but that isn't nice as it can get out of sync

I guess we can continue in this PR fixing compatibility with rack 3

Reply:
I think we should keep an upper limit, Rack 4 (if ever released) might bring breaking changes